validator/helpers/findType.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
```

Comment:
I don't think this and `getAllCardTypes` need to be a separate file. By keeping them in the main file, you don't have to pass in the `customCards` argument.

Reply:
Moved this back into the main file along with getCardPosition